filament/src/materials/blitLow.mat

```diff
@@ -35,7 +35,7 @@ vertex {
 
 fragment {
     void postProcess(inout PostProcessInputs postProcess) {
-#if __VERSION__ == 100
+#if EFFECTIVE_VERSION == 100
```

Comment:
maybe rename to `FILAMENT_EFFECTIVE_VERSION` we have been pretty bad with namespacing, but I've been trying to remedy it.